lib/board.rb

```diff
@@ -0,0 +1,56 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    self.display
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(user_input)
+    index = user_input.to_i - 1
+    @cells[index]
+  end
+
+  def full?
+    @cells.include?(" ") == false
+  end
+
+  def turn_count
+    9 - @cells.count(" ")
+  end
+
+  def taken?(user_input)
+    self.position(user_input) != " "
+  end
+
+  def valid_move?(user_input)
+    user_input.to_i.between?(1,9) && self.taken?(user_input) == false
```

Comment:
You should be able to use the bang operator to utilize negation. This will enable you to avoid comparing to a boolean value and utilize the implicit return value. 

```
user_input.to_i.between?(1,9) && !taken?(user_input)
```